tests/test_cmdline.py

```diff
@@ -429,8 +437,8 @@ def test_list_sources__in_varied_project__lists_all_files(fake_project):
     tmp_path = fake_project(
         files_with_imports={
             "code.py": ["foo"],
-            "subdir/other.py": ["foo"],
-            "subdir/notebook.ipynb": ["foo"],
+            os.path.join("subdir", "other.py"): ["foo"],
+            os.path.join("subdir", "notebook.ipynb"): ["foo"],
```

Comment:
Really only a matter of taste, but since `pathlib` has become available, I've grown allergic to `os.path` 😅 

```suggestion
            str(Path("subdir", "other.py")): ["foo"],
            str(Path("subdir", "notebook.ipynb")): ["foo"],
```